meta-defender-provider/main.go

```diff
@@ -0,0 +1,54 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/http"
+	"os"
+	"strings"
+)
+
+func main() {
+	apiKey := os.Getenv("META_DEFENDER_API_KEY")
+	if apiKey == "" {
+		fmt.Println("META_DEFENDER_API_KEY is not set")
+		os.Exit(1)
+	}
+
+	if len(os.Args) > 1 && os.Args[1] == "validate" {
+		if err := validate(context.Background(), apiKey); err != nil {
```

Comment:
grant changed the framework so that if you are rteurning structure error `{"error": err.string()}` it should print the extra error back to UI without extra error wrapping.

Reply:
Updated.

Reply:
One last thing, it has to exit with 0 so that gptscript will not report extra error wrapping. The framework will catch this special error and throw an error properly.

Reply:
Done. Thanks, Daishan!